TodoList/TodoList/Model/Users.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace TodoList.Data
```

Comment:
```suggestion
namespace TodoList.Models
```
tên folder là `Models` nhé